master/getting-started/kubernetes/troubleshooting.md

```diff
@@ -36,3 +36,17 @@ depend on how your kubelet is configured.  For deployments using `systemd`, you
 
 The log level can be configured via the CNI network configuration file, by changing the value of the
 key `log_level`.  See [the configuration guide]({{site.baseurl}}/{{page.version}}/reference/cni-plugin/configuration) for more information.
+
+#### How do I specify the IP Pools to use to assign pod IPs? 
+
```

Comment:
I'd organize this thusly:

- "Calico IPAM assigns IP addresses from IP Pools, by default, all enabled IP Pools are used"
- "You can specify them in the CNI config - this will apply to all containers started in that CNI network"
- "You can also specify on a per-pod basis in Kubernetes using an annotation, this overrides the CNI config"